packages/core/src/card/InteractableCard.tsx

```diff
@@ -0,0 +1,38 @@
+import { clsx } from "clsx";
+import { ComponentPropsWithoutRef, forwardRef } from "react";
+import { makePrefixer } from "../utils";
+import "./Card.css";
+
+const withBaseName = makePrefixer("saltCard");
+
+export interface InteractableCardProps extends ComponentPropsWithoutRef<"div"> {
+  /**
+   * If `true`, the card will be disabled.
+   */
+  disabled?: boolean;
+}
+
+export const InteractableCard = forwardRef<
+  HTMLDivElement,
+  InteractableCardProps
+>(function Card(props, ref) {
+  const { className, disabled, onClick, children, ...rest } = props;
```

Comment:
should onClick be mandatory? given that we separated this?

Reply:
Maybe not, but just feels like something is missing. It looks like interactable, but might not actually be interactable.

Reply:
I considered this initially too but saw in Button that onClick isn't mandatory...

Reply:
I think the button is different. We always expect a button to have an action on it, but the card is split in two, one with action and one without. 
Something to consider, I guess.

Reply:
tbh I don't quite see the value in separating them

Reply:
it is split in Figma, so for consistency - and it was swayed as it gets confusing having a disabled prop when the component is not necessarily interactive